### DISCUSSION TITLE
Where to put flags

### DISCUSSION THREAD

**Discussion by Jonbivo** [oobabooga/text-generation-webui#4179] [Q&A] [answered]
Hi I just migrated Oobabooga and it says that the old folder is not needed anymore, but in the new folder there isn't webui.py so I don't know where to add flags anymore and writing in the CMD_Flags.txt seem to not do anything

**Comment by berkut1** [comment]
I have no issues with CMD_FLAGS.txt

Make sure you are using the correct flags in CMD_FLAGS.txt.

**Reply by Jonbivo** [reply]
I used the flags from https://github.com/oobabooga/text-generation-webui and it still won't change anything. I fiddled around and found out that to make it work I need to add the flags in line 296 of one_click.rpy

But honestly I feel this was a wasted exercise cause I can't generate replies anymore somehow

**Reply by berkut1** [reply]
show your CMD_FLAGS.txt

**Reply by Jonbivo** [reply]
[CMD_FLAGS.txt](https://github.com/oobabooga/text-generation-webui/files/12819635/CMD_FLAGS.txt)

Here is my CMD_FLAGS.txt

**Reply by berkut1** [reply, marked as answer]
the symbol `#` means - a comment. You should add flags without `#`

**Reply by Jonbivo** [reply]
Ahh yes it worked, thank you!
